contrib/platform/src/com/sun/jna/platform/win32/Psapi.java

```diff
@@ -320,4 +338,79 @@ class PERFORMANCE_INFORMATION extends Structure {
         public DWORD ProcessCount;
         public DWORD ThreadCount;
     }
+
+    @FieldOrder({"Flags"})
+    class PSAPI_WORKING_SET_EX_BLOCK extends Structure {
+        public ULONG_PTR[2] Flags = new ULONG_PTR[Native.POINTER_SIZE == 8 ? 1 : 2];
+        private Long innerValue;
+        private void setInnerValue() {
+            if (innerValue != null) {
+                return;
+            } else if (this.Flags.length == 2) {
+                Long upper = Optional.ofNullable(Flags[1]).map((elem) -> elem.longValue()).orElse(0l);
+                upper = upper << 32;
+                int lower = Optional.ofNullable(Flags[0]).map((elem) -> elem.longValue()).orElse(0l);
+                this.innerValue = upper | lower;
+            } else {
+                this.innerValue = Optional.ofNullable(Flags[0]).map((elem) -> elem.longValue()).orElse(0l);
+            }
+        }
+
+        public Boolean Valid() {
+            setInnerValue();
```

Comment:
(Per above comment if you override `read()` to set `innerValue` you can get rid of these.